test/controllers/patients_controller_test.rb

```diff
@@ -221,7 +224,7 @@ class PatientsControllerTest < ActionDispatch::IntegrationTest
   # confirm sending a 'post' with a payload results in a new patient
   describe 'data_entry_create method' do
     before do
-      @test_patient = attributes_for :patient, name: 'Test Patient'
+      @test_patient = attributes_for :patient, name: 'Test Patient', line_id: create(:line).id
```

Comment:
why'd ya make a new one?

Reply:
I think this was a weird thing about using attributes_for rather than creating the object straight up - iirc it's not quite smart enough to do the auto-population? This solved whatever confusing weird problem I was having.